frontend/src/LogIn/LogIn.vue

```diff
@@ -57,7 +60,22 @@ export default {
         email: this.email,
         password: this.password
       }
-      console.log(target)
+      this.$store.dispatch('auth/login', target)
+        .then(() => {
+          this.msgErr = 'success'
+          this.msgErrColor = 'green'
+          this.$store.dispatch('auth/getProfile')
+            .then(() => { this.msgErr = 'success profile' }) // TODO redirect to profile
+            .catch(err => console.log(err))
+        })
+        .catch(err => {
+          this.msgErrColor = 'red'
+          if (err.status === 400) {
+            this.msgErr = 'wrong password or email'
+          } else {
+            this.msgErr = 'error server'
+          }
+        })
```

Comment:
I think you should use `async`/`await` functions with `try/catch` blocks for a better readability also in the store module too

And I think the vuex auth module should call getProfile actions himself instead of here